src/mavesp8266_component.cpp

```diff
@@ -79,6 +91,14 @@ MavESP8266Component::handleMessage(MavESP8266Bridge* sender, mavlink_message_t*
               return true;
           }
       }
+
+      // recognize FC reboot command and switch to raw mode for bootloader protocol to work
+      if(cmd.target_component == MAV_COMP_ID_ALL
+        && cmd.command == MAV_CMD_PREFLIGHT_REBOOT_SHUTDOWN) {
```

Comment:
Check https://github.com/dogmaphobic/mavesp8266/blob/master/src/mavesp8266_component.cpp#L264 as `MAV_CMD_PREFLIGHT_REBOOT_SHUTDOWN` is processed there. It is used to reboot the ESP8266.